pkg/boot/initrd.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package boot
+
+import (
+	"bytes"
+	"io"
+
+	"github.com/u-root/u-root/pkg/uio"
+)
+
+// PadInitrds pads initrds from a list of io.Readers to a 512 byte boundary
+// and returns a uio.LazyOpenerAt
+func PadInitrds(initrds []io.Reader) *uio.LazyOpenerAt {
```

Comment:
I think it'd be slightly nicer if this was a vararg `PadInitrd(initrds ...io.Reader)`

that way, if you have just one, you don't have to create & alloc the array, and if you have more than one, you can expand it with just ...

```go
PadInitrd([]io.Reader{foo})
PadInitrd(foo) // with Vararg

var readers []io.Reader
PadInitrd(readers)
PadInitrd(readers...) // with vararg
```